catalogData/elasticsearch24/3x-ha/plan.json

```diff
@@ -0,0 +1,6 @@
+{
+  "id": "60defe53-526c-4dd8-aeb1-226d94a7cb48",
+  "name": "3x-ha",
+  "description": "Elasticsearch instance with 3GB of RAM and 1 AWS vCPU",
```

Comment:
Agreed, but I think we should decide on what plans we want to offer before we write the descriptions. Thread here about deciding on plans https://gsa-tts.slack.com/conversation/C0ENP71UG/p1501115222403278